lib/mayaUsd/ufe/UsdUndoMaterialCommands.h

```diff
@@ -178,6 +175,45 @@ class MAYAUSD_CORE_PUBLIC UsdUndoAddNewMaterialCommand : public Ufe::InsertChild
     UsdUndoCreateFromNodeDefCommand::Ptr _createShaderCmd;
 
 }; // UsdUndoAddNewMaterialCommand
+
+//! \brief This command is used to create a materials scope under a specified parent item. A
+//! materials scope is a USD Scope with a special name (usually "mtl"), which holds materials. By
+//! convention, all materials should reside within such a scope.
+class MAYAUSD_CORE_PUBLIC UsdUndoCreateMaterialsScopeCommand
+    : public Ufe::SceneItemResultUndoableCommand
```

Comment:
New command that takes care of creating a materials scope. Factored out from `UsdUndoAssignNewMaterialCommand`. This command will be reused in future work.